src/helm/proxy/clients/huggingface_client.py

```diff
@@ -56,23 +56,20 @@ def __call__(self, input_ids: torch.LongTensor, scores: torch.FloatTensor, **kwa
 class HuggingFaceServer:
     """A thin wrapper around a Hugging Face AutoModelForCausalLM for HuggingFaceClient to call."""
 
-    def __init__(self, pretrained_model_name_or_path: str, revision: Optional[str] = None):
+    def __init__(self, pretrained_model_name_or_path: str, **kwargs):
```

Comment:
Should we add a comment here describing common kwargs that should be specified such as revision, precision, ... ?

Reply:
I would prefer to do this in the docs instead.